lib/beamchmark/formatters/console.ex

```diff
@@ -85,11 +90,25 @@ defmodule Beamchmark.Formatters.Console do
     measurements_diff.context_switches)}
 
     #{entry_header("CPU Usage Average")}
-    #{format_percent_diff(measurements.cpu_info.average_all,
+    #{format_cpu_average(measurements.cpu_info.average_all,
     measurements_diff.cpu_info.average_all)}
+
+
+    #{entry_header("CPU Usage Per Core")}
+    #{format_cpu_by_core(measurements.cpu_info.average_by_core,
+    measurements_diff.cpu_info.average_by_core)}
     """
   end
 
+  defp format_cpu_average(cpu_average_new, cpu_average_diff) do
+    cpu_old = cpu_average_new - cpu_average_diff
+
+    cpu_diff_percent = Math.percent_diff(cpu_old, cpu_average_new)
```

Comment:
This shouldn't be calculated here but in `CpuInfo.diff`. The same for `format_cpu_by_core`